content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md

```diff
@@ -8,11 +8,12 @@ content_template: templates/task
 {{% capture overview %}}
 
 This page explains how to upgrade a Kubernetes cluster created with kubeadm from version
-1.15.x to version 1.16.x, and from version 1.16.x to 1.16.y (where `y > x`).
+1.16.x to version 1.17.x, and from version 1.17.x to 1.17.y (where `y > x`).
 
 To see information about upgrading clusters created using older versions of kubeadm,
 please refer to following pages instead:
 
+- [Upgrading kubeadm cluster from 1.15 to 1.16](https://v1-16.docs.kubernetes.io/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade/)
 - [Upgrading kubeadm cluster from 1.14 to 1.15](https://v1-15.docs.kubernetes.io/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-15/)
 - [Upgrading kubeadm cluster from 1.13 to 1.14](https://v1-15.docs.kubernetes.io/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-14/)
 - [Upgrading kubeadm cluster from 1.12 to 1.13](https://v1-15.docs.kubernetes.io/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-13/) or [Upgrading kubeadm HA clusters from v1.12 to v1.13](https://v1-15.docs.kubernetes.io/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-ha-1-13/)
```

Comment:
12->13 and 13->14 are outside of the support skew with the release of 1.17.
keeping them here is to help the users, but not ideal.

Reply:
I think that keeping one more than the official version skew is ok, but after that the user can simply switch to older version of the docs site